src/components/DragDrop/DragDrop.tsx

```diff
@@ -78,60 +77,63 @@ export const DragDrop = ({
   /**
    * Target drag drop and drag drop inner DOM elements
    */
-  const dragDropInnerRef = useRef() as MutableRefObject<HTMLDivElement>;
-  const dragDropRef = useRef() as MutableRefObject<HTMLElement>;
+  const dragDropInnerRef = useRef<HTMLDivElement>(null);
+  const dragDropRef = useRef<HTMLElement | null>(null);
   /**
    * Set right and left gradients on tables
    */
   const setShadows = () => {
-    /**
-     * Target the actual drag drop inside drag drop.
-     */
-    const overflowListItems = Array.from(dragDropInnerRef.current.children);
-    let childrenWidth = 0;
-
-    overflowListItems.forEach(function (item) {
-      const itemMarginLeft = parseInt(
-        getComputedStyle(item).marginLeft.slice(0, -2),
-      );
-      childrenWidth += item.clientWidth + itemMarginLeft;
-    });
-    if (overflowListItems) {
+    if (dragDropInnerRef.current) {
       /**
-       * Get the width of the drag drop offscreen when drag drop overflows.
+       * Target the actual drag drop inside drag drop.
        */
-      const dragDropWidth = dragDropRef.current.clientWidth;
-      const totalItemsWidth = childrenWidth;
-      const dragDropOffScreen =
-        totalItemsWidth - dragDropRef.current.clientWidth;
+      const overflowListItems = Array.from(dragDropInnerRef.current.children);
+      let childrenWidth = 0;
 
-      if (totalItemsWidth <= dragDropWidth) {
-        /**
-         * If drag drop width is less than or equal to drag drop width, remove all shadows
-         */
-        setIsEnd(false);
-        setIsStart(false);
-      } else if (
-        dragDropInnerRef.current.scrollLeft > 0 &&
-        dragDropInnerRef.current.scrollLeft < dragDropOffScreen
-      ) {
-        /**
-         * If drag drop inner scroll isn't all the way to the left or to the right, turn all shadows on
-         */
-        setIsEnd(true);
-        setIsStart(true);
-      } else if (dragDropInnerRef.current.scrollLeft >= dragDropOffScreen) {
-        /**
-         * If drag drop inner scroll is >= to width of drag drop offscreen, turn off right shadow and turn left shadow on
-         */
-        setIsEnd(false);
-        setIsStart(true);
-      } else {
+      overflowListItems.forEach(function (item) {
+        const itemMarginLeft = parseInt(
+          getComputedStyle(item).marginLeft.slice(0, -2),
+        );
+        childrenWidth += item.clientWidth + itemMarginLeft;
+      });
+
+      if (dragDropRef.current) {
```

Comment:
May be hard to tell in the diff, but I've removed the

```ts
if (overflowListItems) {
```

check. That array will always be truthy, so it's not necessary.

Replaced with this

```ts
if (dragDropRef.current) {
```

which is needed because we have to do the `null` check now.